pytest_rpc/helpers.py

```diff
@@ -252,3 +253,26 @@ def parse_swift_ring_builder(ring_builder_output):
             swift_data[k] = float(v)
 
     return swift_data
+
+
+def get_cinder_major_version(run_on_host):
+    """ Retrieve cinder version number from utility container
+
+    Args:
+        run_on_host (testinfra.host.Host): Testinfra host fixture
+
+    Return:
+        int: cinder major version, 0 if lookup fails
+
+    """
+
+    cmd = '. openrc ; cinder --version'
+    result = run_on_container(cmd, 'utility', run_on_host)
+
+    try:
+        v = Version(result.stdout)
+        major = v.release[0]
+    except (InvalidVersion):
+        major = 0
```

Comment:
Just a nitpick: wondering if we can assign `major` to be a non integer here? so that when we call the function using a conditional like `if major < 2`and the cinder version is invalid it will return errors.
Of course we can use the condition `if 0 < major < 2` but I am thinking about preventing user error.

Reply:
@phongdly this is certainly an option. My thought here was that in the event that the cinder version cannot be determined I would leave the decision up to the caller about how to handle an invalid version. Since '0' is an invalid version for our domain, the caller can choose to test for a version > 0 *or*, as I have done, attempt to validate the generic `cinder` service regardless of the invalid version.
Perhaps `0` does not make this intention clear. I could change it to `-1` to make it more explicit.

Reply:
👍 for `-1`